README.md

```diff
@@ -24,11 +24,10 @@ DataLad Catalog is a free and open source command line tool, with a Python API,
 
 ### Acknowledgements
 
-This software was developed with support from the German Federal Ministry of
-Education and Research (BMBF 01GQ1905), the US National Science Foundation
-(NSF 1912266), and the Deutsche Forschungsgemeinschaft (DFG, German Research
-Foundation) under grant SFB 1451 ([431549029](https://gepris.dfg.de/gepris/projekt/431549029),
-INF project).
+This software was developed with support from:
+- the German Federal Ministry of Education and Research (BMBF 01GQ1905), the US National Science Foundation
+(NSF 1912266), and the Deutsche Forschungsgemeinschaft (DFG, German Research Foundation) under grant SFB 1451 ([431549029](https://gepris.dfg.de/gepris/projekt/431549029), INF project).
+- the [MKW-NRW: Ministerium für Kultur und Wissenschaft des Landes Nordrhein-Westfalen](https://www.mkw.nrw/) under the Kooperationsplattformen 2022 program, grant number: KP22-106A
```

Comment:
```suggestion
This software was developed with support from:
- the German Federal Ministry of Education and Research (BMBF 01GQ1905)
- the US National Science Foundation
(NSF 1912266)
- the Deutsche Forschungsgemeinschaft (DFG, German Research Foundation) under grant SFB 1451 ([431549029](https://gepris.dfg.de/gepris/projekt/431549029), INF project).
- the [MKW-NRW: Ministerium für Kultur und Wissenschaft des Landes Nordrhein-Westfalen](https://www.mkw.nrw/) under the Kooperationsplattformen 2022 program, grant number: KP22-106A
```

If we do a list, let's have each grant be an item.

Reply:
Oops, I missed that, thanks!